### ISSUE TITLE
don't check the critical-load at cut-over section

### ISSUE THREAD

**Issue by rj03hou** [github/gh-ost#329]
I check the 1.0.30 release note, find that in this version we find fix a bug "Fixed cut-over stall", but we meet another similar case:

```
2016-12-07 19:12:17 INFO Creating magic cut-over table `test`.`_test_del`
2016-12-07 19:12:17 INFO Magic cut-over table created
2016-12-07 19:12:17 INFO Locking `test`.`test`, `test`.`_test_del`
2016-12-07 19:12:17 INFO Tables locked
2016-12-07 19:12:17 INFO Session locking original & magic tables is 2164861
2016-12-07 19:12:17 INFO Writing changelog state: AllEventsUpToLockProcessed:1481109137528939398
2016-12-07 19:12:17 INFO Intercepted changelog state AllEventsUpToLockProcessed
2016-12-07 19:12:17 INFO Handled changelog state AllEventsUpToLockProcessed
2016-12-07 19:12:17 INFO Waiting for events up to lock
2016-12-07 19:12:18 FATAL critical-load met: Threads_running=54, >=32
```

means critical-load check may happens in the cut-over section, when we lock the tables, the  Threads_running may increase quickly but shortly, so I think we should not check the critical-load in the cut-over section.

I try to fix this, but it not  easy to reproduction, I modify the code to reproduce.
```go
//applier.go
func (this *Applier) makeTestLoad() {
	for {
		sqlutils.ExecNoPrepare(this.db, `select count(*) from test`)
	}

}

//migrator.go add the following code to atomicCutOver method
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()
	go this.applier.makeTestLoad()

	time.Sleep(2 * time.Second)

	//lockOriginalSessionId := <-lockOriginalSessionIdChan
```

and set the parameter
--max-load=Threads_running=10 \
--critical-load=Threads_running=10 \

to reproduce the problem. And I think we should cleanup if we meet a panic abort.

**Comment by shlomi-noach** [comment]
This issue has been considered in https://github.com/github/gh-ost/issues/216, and solved by https://github.com/github/gh-ost/pull/266

TL;DR: there's a reason for `--critical-load`; ignoring it altogether is not the right thing to do.

`--critical-load-interval-millis` provides with a reasonable solution: define it to, say, `3` seconds (which is basically the `cut-over` timeout anyhow), and so the app will only bail out if `critical-load` is met _again_ after `3` seconds.

**Comment by rj03hou** [comment]
--critical-load-interval-millis is good, set this can solve our problem.
next time I will first discuss in the issue, then try to write the code.

**Comment by shlomi-noach** [comment]
@rj03hou thank you very much for your continued contributions!
